Gemfile.lock

```diff
@@ -646,7 +643,7 @@ GEM
     xpath (2.0.0)
       nokogiri (~> 1.3)
     zonebie (0.6.1)
-    zxcvbn-js (4.4.1)
+    zxcvbn-js (4.4.2)
```

Comment:
Have we spot-checked this to make sure our `zxcvbn_overrides.rb` still points to the right file?